src/plugins/intel_cpu/src/nodes/space_to_batch.cpp

```diff
@@ -2,26 +2,21 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
-#include <cmath>
-#include <vector>
-#include <string>
-#include <dnnl_types.h>
-#include "ie_parallel.hpp"
-#include "utils/bfloat16.hpp"
-#include <selective_build.h>
 #include "space_to_batch.h"
+
+#include "ie_parallel.hpp"
 #include <nodes/common/blocked_desc_creator.h>
-#include <ngraph/opsets/opset2.hpp>
+#include <openvino/op/ops.hpp>
```

Comment:
why not to include only headers with required ops?
Including everything increases compilation time

Reply:
Agree. Modified